prow/cluster/kubernetes-external-secrets_crd.yaml

```diff
@@ -1,6 +1,5 @@
 ---
-# From https://github.com/external-secrets/kubernetes-external-secrets/tree/f866e34e0319d9c54ea17d07f5d5818a28d9d0f6
```

Comment:
Please include a line like this indicating where the new file was copied from.

Reply:
It wasn't; that version doesn't seem to have a v1.22 compatible copy.

This was created by following https://kubernetes.io/docs/reference/using-api/deprecation-guide/#v1-22 instead

Reply:
What about this? https://github.com/external-secrets/kubernetes-external-secrets/blob/master/charts/kubernetes-external-secrets/crds/kubernetes-client.io_externalsecrets_crd.yaml

Reply:
The link posted by Cole should work, according to release notes https://github.com/external-secrets/kubernetes-external-secrets/releases/tag/8.0.1 has updated to crd/v1

Reply:
Copied and added line